soletta/builder.go

```diff
@@ -0,0 +1,42 @@
+package soletta
+
+/*
+#include <soletta.h>
+#include <sol-flow-builder.h>
+*/
+import "C"
+import "unsafe"
+
+type FlowBuilder struct {
+	builder *C.struct_sol_flow_builder
+}
+
+func (fb *FlowBuilder) Init() {
+	fb.builder = C.sol_flow_builder_new()
+}
```

Comment:
I think this could be more idiomatic. To my mind would be better a constructor-like function that instantiate the object and automatically call  `sol_flow_builder_new()`.

```
func NewBuilder() FlowBuilder {
         builder := FlowBuilder{}
         builder.Init()
        return builder
}
```